source/renderer/app/components/staking/redeem-itn-rewards/Step3FailureDialog.scss

```diff
@@ -2,18 +2,24 @@
   display: block;
   margin-bottom: 30px;
   text-align: center;
+
   svg {
+    fill: transparent;
```

Comment:
This doesn't seem to be needed

Reply:
Its needed because overall svg has transparent color and its child elements have fill or stroke depending on the icon.